styles.css

```diff
@@ -63,3 +63,40 @@ form {
   font-size: 24pt
 }
 
+.fa-arrow-circle-down,
+.fa-arrow-circle-right {
+  margin-top: 20px;
+  font-size: 24pt;
+}
+
+span {
```

Comment:
Try to avoid styling elements unless you absolutely need to. Instead, define a custom class with a meaningful name and use that.